webapp/src/pages/applicationForm/components/ApplicationForm.js

```diff
@@ -1111,9 +1110,11 @@ class ApplicationForm extends Component {
         isLoading: false,
         selectedResponse: selectFirstResponse ? responseResponse.response[0] : null,
         responseSelected: selectFirstResponse,
-        event: eventResponse.event
+        event: eventResponse.event,
+        journalSubmissionFlag: this.props.journalSubmissionFlag
```

Comment:
Why add this to the state? It doesn't seem like it's being changed, so you can just refer to it from this.props whenever you need it.

Reply:
I forgot to delete it. I believe I was implementing a different logic at that time.